megamek/src/megamek/client/bot/CEntity.java

```diff
@@ -264,15 +265,15 @@ public void characterize() {
         bv = entity.calculateBattleValue();
 
         // Make a guess as to whether MASC should be turned on or off
-        // TODO: Link this to a Bot configuration file
+        // TODO : Link this to a Bot configuration file
         runMP = entity.getRunMP();
         if (entity instanceof Mech) {
-            Entity.MPBoosters mpBoosters = ((Mech) entity).getMPBoosters();
-            if (mpBoosters.hasMASCAndOrSupercharger()) {
+            MPBoosters mpBoosters = entity.getMPBoosters();
+            if (!mpBoosters.isNone()) {
```

Comment:
Is there an opposite version of this method so negation doesn't need to be done? For example a `isAvailable()` method?

Reply:
There is not.

Reply:
Skimming through this review I see there is a way to add one. From a readability stand point (I skipped over the `!` about half the time is why I'm asking this) it would be good to add.